File: src/main/java/me/centralhardware/znatoki/telegram/statistic/telegram/CommandHandler/organization/GrafanaCommand.java
package me.centralhardware.znatoki.telegram.statistic.telegram.CommandHandler.organization;

import lombok.RequiredArgsConstructor;
import me.centralhardware.znatoki.telegram.statistic.entity.postgres.Role;
import me.centralhardware.znatoki.telegram.statistic.mapper.postgres.OrganizationMapper;
import me.centralhardware.znatoki.telegram.statistic.mapper.postgres.UserMapper;
import me.centralhardware.znatoki.telegram.statistic.telegram.CommandHandler.CommandHandler;
import org.springframework.stereotype.Component;
import org.telegram.telegrambots.meta.api.objects.Message;

@Component
@RequiredArgsConstructor
public class GrafanaCommand extends CommandHandler {

    private final UserMapper userMapper;
    private final OrganizationMapper organizationMapper;

    @Override
    public void handle(Message message) {
        var user = userMapper.getById(message.getFrom().getId());
        if (user.getRole() != Role.ADMIN) {
            sender.sendText("Доступ запрещен", message.getFrom());
            return;
        }

        var org = organizationMapper.getById(user.getOrganizationId());
        sender.sendText(STR."""
                адрес: \{org.getGrafanaUrl()}
                пользователь: \{org.getGrafanaUsername()}
                пароль: \{org.getGrafanaPassword()}
                """, message.getFrom());
    }

    @Override
    public boolean isAcceptable(String data) {
        return data.equalsIgnoreCase("/grafana");
    }
}

File: src/main/kotlin/me/centralhardware/znatoki/telegram/statistic/telegram/commandHandler/organization/GrafanaCommand.kt
package me.centralhardware.znatoki.telegram.statistic.telegram.commandHandler.organization

import dev.inmo.tgbotapi.extensions.api.send.sendMessage
import dev.inmo.tgbotapi.types.message.abstracts.CommonMessage
import dev.inmo.tgbotapi.types.message.content.TextContent
import me.centralhardware.znatoki.telegram.statistic.bot
import me.centralhardware.znatoki.telegram.statistic.mapper.ConfigMapper

suspend fun grafanaCommand(message: CommonMessage<TextContent>){
    bot.sendMessage(
        message.chat,
        """
            адрес: ${ConfigMapper.grafanaUrl()}
            пользователь: ${ConfigMapper.grafanaUsername()}
            пароль: ${ConfigMapper.grafanaPassword()}
        """.trimIndent())
}